app/assess/routes.py

```diff
@@ -66,9 +66,11 @@ def display_sub_criteria(
     theme_id = args.get("theme_id", sub_criteria.themes[0].id)
     fund = get_fund(Config.COF_FUND_ID)
     comments = get_comments(
-        application_id=application_id, sub_criteria_id=sub_criteria_id
+        application_id=application_id, sub_criteria_id=sub_criteria_id, theme_id=theme_id
     )
 
+    comments_dictionary = {theme.id: [comment for comment in comments if comment.theme_id == theme.id] for theme in sub_criteria.themes}
```

Comment:
Little suggestion, for dictionaries I usually go with an `x_to_y` naming convention, so people typically know what's being mapped without having to read the dict comprehension, for this I'd prob go with: `theme_id_to_comments_list_map`